ui/app/router.js

```diff
@@ -17,6 +17,15 @@ Router.map(function () {
       this.route('dashboard');
       this.mount('config-ui');
       this.mount('sync');
+      // TODO remove conditional once further feature work for single item recovery for release 1.21 is completed
+      if (config.environment !== 'production') {
+        this.route('recovery', function () {
+          this.route('snapshots', function () {
```

Comment:
[Optional] - It's 100% superflous, but I think it's nice to explicitly call out the index route in the router. I know it'd be diverent pattern in this file, but it's nice to see that there's _something_ there at the default `/`


```suggestion
          this.route('snapshots', function () {
            this.route('index', { path: '/' });
```